### PR TITLE
feat: Add a bunch o' new panels to the homepage.

### PR DESCRIPTION
**Pull request by sarahmonster**
Needs fleshing out & fine-tuning, obviously, but this is just about getting the new structure in place.

This is what it should (sort of) look like when it's finished:
Mockup of octopusthink.com homepage, showing lots of different panels of content.

Notes & commentary on #105.

@tofumatt most of this work is going to be a bunch of CSSing/design work that I can handle, but I'd like your help with three things:

- [x] Pulling the two most recent blog posts into the blog panel.
- [x] Showing three portfolio items in the portfolio panel. (Ideally this would be either random selection, or numbers defined somewhere in the homepage MDX. Failing those options, using portfolio item frontmatter is acceptable.
- [ ] Copyedits & content-thoughts for the final panel ("Work with us"). 

I'll be asking for more fine-grained review as I pull everything together, but those are the salient points here.

### CODE REVIEW

**Comment by tofumatt** [comment]
I've added the portfolio and the blog components that pull in dynamic data to the homepage. I've not done the copy-edits yet, but I think the blog posts bit will need some styling 😄

**Comment by sarahmonster** [comment]
Did some styling tweaks! Still could use refining:

- [ ] consider one CTA (rather than two) for blog panel
- [ ] accessibility check
- [ ] fine-tune logos on smaller devices (requires adding mobileColumns/tabletColumns/desktopTablets etc props to GridLayout, like panel)
- [ ] hide second portfolio item on tablets
- [ ] shorten services text on mobile
- [ ] replace about photo
- [ ] replace constructopus with an illustration
- [ ] copyedit "work with us" text

**Comment by sarahmonster** [comment]
I rebased and force-pushed, don't hate me!

I've fixed as many of the accessibility issues flagged by Axe as I'm able to/can figure out, but I'm not certain about some of them. 

What's left for me to do:
- Fine-tuning the pinks & cyans to ensure they have sufficient contrast (finicky, but in progress)
- Ensure portfolio items have some kind of focus state (can be done as an add-on, I suppose)

I'm having to manually edit out ids from svgs, so we definitely want svgo (#48) at some point, espeically if we want to do more svg work.

Anyway. I think this is probably about ready to review/merge now.

File: src/components/FriendLogos/index.js [hunk 1]
```diff
@@ -3,37 +3,41 @@ import React from 'react';
...
-      <GridLayout columns="3" gap="8.0">
-        <Image src="standard.png" alt="Standard Broadcast logo" />
+      <GridLayout columnsMobile="2" columnsTablet="3" gap="8.0">
+        {!abbreviated && <Image src="standard.png" alt="Standard Broadcast logo" />}
         <Image src="firefox.png" alt="Firefox logo" />
-        <Image src="wordpress.com.png" alt="WordPress.com logo" />
+        {!abbreviated && <Image src="wordpress.com.png" alt="WordPress.com logo" />}
         <Image src="snp.png" alt="Scottish National Party logo" />
-        <Image src="ns-government.png" alt="Nova Scotia Government logo" />
+        <Hide mobile>
```
Comment (tofumatt) [inline src/components/FriendLogos/index.js]: What is happening here? Why is this hidden on mobile at all times? Should this only be hidden when in abbreviated mode or on both pages? This seems a wee bit fragile, and just want to double-check the intention here.

Comment (tofumatt) [reply src/components/FriendLogos/index.js]: I figured out what's happened by running the code and this causes an uneven number of items on mobile on the _main_ portfolio page:

Screenshot 2020-01-23 19 38 31

It seems what you meant was `!abbreviated ? <Image src="ns-government.png" alt="Nova Scotia Government logo" /> : (<Hide mobile><Image src="ns-government.png" alt="Nova Scotia Government logo" /></Hide>)`

I don't love it, but it'll do. I've made the change.

**Pull request merged**